include/translatable-object-with-types-trait.php

```diff
@@ -46,12 +32,14 @@ protected function get_objects_with_no_lang_sql( array $language_ids, $limit, ar
 			$args = $this->get_translated_object_types();
 		}
 
+		$db = $this->get_db_infos();
```

Comment:
It's weird to me the trait is aware of this method. Do you think it should implement it?